src/util/fork_utils.js

```diff
@@ -38,11 +38,22 @@ function start_workers(metrics_port, count = 0) {
         // We don't want to leave the process with a partial set of workers,
         // so if any worker exits, we will print an error message in the logs and start a new one.
         cluster.on('exit', (worker, code, signal) => {
-            console.warn('WORKER exit', { id: worker.id, pid: worker.process.pid, code, signal }, 'starting a new one.');
+            console.warn('WORKER exit', { id: worker.id, pid: worker.process.pid, code, signal });
             new NoobaaEvent(NoobaaEvent.FORK_EXIT).create_event(undefined, { id: worker.id, pid: worker.process.pid,
                 code: code, signal: signal}, undefined);
+            // SIGABRT is used for abnormal program termination - https://www.qnx.com/developers/docs/6.4.1/dinkum_en/ecpp/stdlib.html
+            // we saw it happen both in case of no memory and on too much processes started
```

Comment:
what about other signals? I remember we talked about if we see 2 panics in 10 minutes we will also exit the main process

Reply:
Not relevant anymore - we changed the solution to exit on too many fork failures.